src/sentry/release_health/sessions.py

```diff
@@ -17,6 +18,7 @@
     _check_releases_have_health_data,
     _get_release_adoption,
     _get_release_sessions_time_bounds,
+    get_crash_free_breakdown,
```

Comment:
please rename this function to underscore, gives me some assurance that we have ported all usages over to the service